data/transactions/logic/assembler.go

```diff
@@ -479,7 +483,7 @@ func assembleTxn(ops *OpStream, args []string) error {
 	}
 	val, ok := txnFields[args[0]]
 	if !ok {
-		return fmt.Errorf("txn unknown arg %v", args[0])
+		return fmt.Errorf("txn unknown arg %s", args[0])
```

Comment:
Now I look at this and actually I wanted %#v so that it'll do quoting and escaping if someone put in a non-trivial string (same below in assembleGtxn)